Gemfile

```diff
@@ -1,8 +0,0 @@
-source "https://rubygems.org"
```

Comment:
Please add this file back, thanks.

Reply:
Any particular reason why you guys prefer using `Gemfile` rather than using the `_config.yml` for testing locally? (Using Gemfile causes a lot of issues for the end users like me who might have a more recent version of Gems than the ones specified in Gemfile.

If you are fine with this change, I will remove the `Gemfile` line from `_config.yml` else I will revert it back but then you will see a lot of issues like #951 in future.

Reply:
> Does `_config.yml` allow for groups and specific sources?

I am not sure about groups, but I do think we can specify specific sources. 

Now I checked the removed file, seems like you guys have a test environment, I will try to experiment locally if specifying a version works.